apps/web/app/components/RichText/Editor.tsx

```diff
@@ -28,6 +29,12 @@ const Editor = ({ content, placeholder, onChange, id, className }: EditorProps)
     },
   });
 
+  useEffect(() => {
+    if (editor && content !== editor.getHTML()) {
+      editor.commands.setContent(content || "");
+    }
+  }, [content]);
```

Comment:
React Hook useEffect has a missing dependency: 'editor'. Either include it or remove the dependency array

if intended you can disable this warning with note why